ee/hogai/graph/insights/nodes.py

```diff
@@ -112,6 +112,8 @@ class InsightSearchNode(AssistantNode):
     INSIGHTS_CUTOFF_DAYS = 180
     MAX_SERIES_TO_PROCESS = 3
 
+    REASONING_MESSAGE_CONTENT = "Searching for insights"
```

Comment:
It doesn't seem to be used somewhere, wrong name? 👀

Reply:
Great catch!